catroid/src/main/java/org/catrobat/catroid/content/actions/SetNfcTagAction.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2016 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.content.actions;
+
+import android.nfc.NdefMessage;
+import android.util.Log;
+
+import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
+
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.formulaeditor.Formula;
+import org.catrobat.catroid.formulaeditor.InterpretationException;
+import org.catrobat.catroid.nfc.NfcHandler;
+import org.catrobat.catroid.stage.StageActivity;
+
+public class SetNfcTagAction extends TemporalAction {
+
+	private Sprite sprite;
+	private Formula nfcNdefMessage;
+	private int spinnerSelection;
```

Comment:
Please rename everywhere.

The selected item defines something. e.g. the selected item in the [BroadcastBrick](https://github.com/Catrobat/Catroid/blob/1866ce072f4b4fa398f7caae8411eb903bff4bf1/catroid/src/main/java/org/catrobat/catroid/content/bricks/BroadcastBrick.java#L49) defines the [broadcastMessage](https://github.com/Catrobat/Catroid/blob/1866ce072f4b4fa398f7caae8411eb903bff4bf1/catroid/src/main/java/org/catrobat/catroid/content/bricks/BroadcastBrick.java#L49).